cmd/lncli/cmd_walletunlocker.go

```diff
@@ -495,7 +495,7 @@ func unlock(ctx *cli.Context) error {
 	// echoed in the console.
 	case ctx.IsSet("stdin"):
 		reader := bufio.NewReader(os.Stdin)
-		pw, err = reader.ReadBytes('\n')
+		pw, err = ioutil.ReadAll(reader)
```

Comment:
Below we trim newlines, seems like that could be an issue if a password ends with a newline?